tools/installer/BUILD.gn

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Simplified version based on //chrome/installer/linux
+import("//xwalk/build/version.gni")
+
+action("xwalk_installer_package_deb") {
+  packaging_files_binaries = [ "$root_out_dir/xwalk" ]
+  if (is_component_ffmpeg) {
+    packaging_files_binaries += [ "$root_out_dir/libffmpeg.so" ]
+  }
+
+  script = "//chrome/installer/linux/flock_make_package.py"
+  if (current_cpu == "x86") {
+    # The shell scripts use "ia32" instead of "x86".
+    build_script_arch = "ia32"
+  } else {
+    build_script_arch = current_cpu
+  }
+
+  if (current_cpu == "x86") {
+    deb_arch = "i386"
+  } else if (current_cpu == "x64") {
+    deb_arch = "amd64"
+  } else if (current_cpu == "arm") {
```

Comment:
I wouldn't worry about the "arm" case for now: it's not in the gyp version and we've never tried to create an ARM Debian package so it's not even clear if it works.